src/main/resources/application-postgres.yaml

```diff
@@ -0,0 +1,6 @@
+spring:
+  datasource:
+    url: jdbc:postgresql://localhost:5432/prison-estate-db
+    username: admin
+    password: admin_password
```

Comment:
I remember we used to use some security scanning tool that used to pick up on things like this - it would think you are adding a production password to git - it was so annoying.